install_config/adding_hosts_to_installs.adoc

```diff
@@ -0,0 +1,195 @@
+[[install-config-adding-hosts-to-installs]]
+= Adding Hosts to an Existing Install
```

Comment:
I think we should stick with calling it an "Existing Cluster" instead of an "install". IBM Style Guide says to only use "install" as a verb, too. "Installation" is permitted as a noun, but I still think sticking with "cluster" is more specific, and (considering consistency) we have a lot of other spots that refer to it in that way currently.